fairworkflows/fairstep.py

```diff
@@ -13,15 +13,15 @@
 from fairworkflows.rdf_wrapper import RdfWrapper
 
 FAIRSTEP_PREDICATES = [RDF.type, namespaces.PPLAN.hasInputVar,
```

Comment:
This is not used anywhere anymore, I removed it in a just merged PR so you'll encounter it when resolving merge conflicts

Reply:
OK! Removed